internal/github/checkUnsigned.go

```diff
@@ -133,3 +141,30 @@ func CheckAndComment(ctx context.Context, client *github.Client, owner, repo str
 
 	return nil
 }
+
+func checkAndRemoveComment(ctx context.Context, client *github.Client, owner, repo string, prNumber int) error {
+	comments, _, err := client.Issues.ListComments(ctx, owner, repo, prNumber, nil)
+	if err != nil {
+		return fmt.Errorf("error fetching comments: %v", err)
+	}
+
+	// Check if the comment exists
+	for _, comment := range comments {
+		if comment.GetUser().GetLogin() == automationBotName && strings.EqualFold(comment.GetBody(), unsignedCommitMessage) {
+			commentID := comment.GetID()
+			slogs.Logr.Info("Found unsigned commit comment to remove",
+				"comment_id", commentID,
+				"author", automationBotName,
+				"pr_number", prNumber)
+			_, err := client.Issues.DeleteComment(ctx, owner, repo, commentID)
+			if err != nil {
+				return fmt.Errorf("error deleting comment %d: %v", commentID, err)
```

Comment:
```suggestion
				return fmt.Errorf("error deleting comment %d: %w", commentID, err)
```

%w to wrap errors - %v just dumps struct data